src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -382,7 +382,10 @@ class TaskForm extends React.Component<Props, State> {
                             className={pillSelectorOverlayClasses}
                             error={this.getErrorByFieldname('taskAssignees')}
                             disabled={isForbiddenErrorOnEdit}
-                            inputProps={{ 'data-testid': 'task-form-assignee-input' }}
+                            inputProps={{
+                                'data-testid': 'task-form-assignee-input',
+                                'data-target-id': 'CreateTaskModal-selectAssigneesInput',
```

Comment:
I see from the other comment that the structure is: `[elementName]-[identifier]`

how was `CreateTaskModal` determined as the element name in this case?

Reply:
@kkuliczkowski-box said that when we add ids manually then we can create meaningful names.
Wdyt @kkuliczkowski-box?

Reply:
I also think that using the `elementName` as the first part should be maintained.